xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/server/ServerInfoLedger.java

```diff
@@ -67,15 +67,15 @@ static ImmutableServerInfoLedger.Builder builder() {
    * @return An {@link UnsignedInteger} representing the amount of XRP to reserve.
    */
   @JsonProperty("reserve_base_xrp")
-  UnsignedInteger reserveBaseXrp();
+  BigDecimal reserveBaseXrp();
```

Comment:
This is a breaking change. We need to introduce the other format that @nkramer44 posted about in slack.